timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/rdf/tripleprocessor/PersonNameVariantTripleProcessorTest.java

```diff
@@ -24,7 +24,7 @@
 
 public class PersonNameVariantTripleProcessorTest {
   private static final String PERSON_URI = "http://example.com/Jan";
-  private static final String TIM_IS_NAME_VARIANT_OF = "http://timbuctoo.com/isNameVariantOf";
+  private static final String TIM_IS_NAME_VARIANT_OF = "http://timbuctoo.huygens.knaw.nl/isNameVariantOf";
```

Comment:
This constant is also declared in TripleProcessorImpl.java and it seems that both are the same thing. an import in one place or the other would be better.